devmgmtui/src/components/cloud/CloudDownload.js

```diff
@@ -320,8 +321,6 @@ class CloudDownload extends Component {
 
 	componentDidMount() {
 		document.title = 'Cloud Download';
-		
```

Comment:
Why is this removed ?

Reply:
It is stylistic. The space looked odd, nothing significant.